packages/core/application/core-application-browser-internal/src/application_service.tsx

```diff
@@ -346,9 +346,17 @@ export class ApplicationService {
       navigateToApp,
       navigateToUrl: async (
         url: string,
-        { skipAppLeave = false, forceRedirect = false, state }: NavigateToUrlOptions = {}
+        {
+          skipAppLeave = false,
+          forceRedirect = false,
+          openInNewTab = false,
+          state,
+        }: NavigateToUrlOptions = {}
       ) => {
         const appInfo = parseAppUrl(url, http.basePath, this.apps);
+        if (openInNewTab && appInfo) {
+          return this.openInNewTab!(getAppUrl(availableMounters, appInfo.app, appInfo.path));
+        }
```

Comment:
Why do you need `appInfo` to be resolved to open in a new tab? we already have the url passed as parameter, we don't need more to call `this.openInNewTab`, do we?